orm/table.go

```diff
@@ -188,8 +203,13 @@ func (a Table) Delete(ctx HasKVStore, rowID RowID) error {
 	return nil
 }
 
-// Has checks if a key exists. Panics on nil key.
-func (a Table) Has(ctx HasKVStore, rowID RowID) bool {
+// Has checks if a key exists. Panics on nil or empty key.
```

Comment:
not sure that this comment is true anymore - looks like it will just return false on the empty key

Reply:
True - we accept empty key, but panic on `nil`.

Reply:
the nil value here would just be the empty []byte slice though, so there shouldn't ever be a panic -->> https://play.golang.org/p/eLzItdNL8He

Reply:
right, will do a PR to update the comment.